.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+### Link to JIRA
+
+[PLAT-XYZ](https://dominodatalab.atlassian.net/browse)
```

Comment:
Cosmetic: change to DOM-XYZ. PLAT JIRA project is typically used by the Distributions team.